homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
 SERVICE_SYNC = 'sync'
 
+DEFAULT_NUM_REPEATS = '1'
+DEFAULT_DELAY_SECS = '0.4'
```

Comment:
Harmony hub remotes have built-in delays for each command. So each-command would be 400ms + Harmony's configured delay (for ex. mine is set to 300ms). So for each vol-up it'd be 700ms which would still be slower than using a physical remote.

Is there a way to change this delay value within configuration.yaml, also, have you tested it with a lower configured delay? I'm assuming HA is multi-threaded so it could handle these commands.